src/cdk/v2/destinations/linkedin_ads/data/linkedinUserInfoConfig.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "destKey": "firstName",
+    "sourceKeys": "firstName",
+    "required": false,
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "lastName",
+    "sourceKeys": "lastName",
+    "required": false,
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "title",
+    "sourceKeys": "title",
+    "required": false,
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "companyName",
+    "sourceKeys": "context.traits.companyName",
+    "required": false
+  },
```

Comment:
```suggestion
  {
    "destKey": "companyName",
    "sourceKeys": ["traits.companyName", "context.traits.companyName"],
    "required": false
  },
```

Reply:
this is track call, normal traits are not supposed to be present

Reply:
This is a track call, `message.traits` are not expected to be here